src/spec/payid-easy-checkout.md

```diff
@@ -0,0 +1,369 @@
+
+# Introduction
+
+The PayID Easy Checkout Protocol is a minimal protocol that allows a recipient (e.g., an online merchant or a charity) to
+request a payment from a payer using only the payer's PayID. Implementations
+of the protocol should require little to no server-side engineering efforts, while creating an improved and uniform
+user experience for payers.
+
+The main focus of the Protocol is on PayID Easy Checkout Discovery, which defines how a PayID client can use a PayID
+to retrieve a PayID Easy Checkout URL which represents a resource that the payer's digital wallet can use to initiate 
+a payment to the merchant. 
+
+Though the [appendix](#appendix) of this specification provides an example usage of a 
+PayID Easy Checkout URL using Web Redirects, supplemental RFCs are needed to define the different ways in which a PayID
+client can utilize a PayID Easy Checkout URL.
+
+# PayID Easy Checkout Protocol
+The PayID Easy Checkout Protocol can be used to facilitate an end-to-end checkout flow between a payment recipient, such
+as an online merchant, and a sending client, such as a wallet.
+
+The protocol is comprised of two parts:
+
+1. PayID Easy Checkout Discovery
+2. PayID Easy Checkout URL Assembly
+
+## PayID Easy Checkout Discovery
+PayID Easy Checkout Discovery extends [PAYID-DISCOVERY][] by defining a new link in the PayID metadata JRD
+returned by a PayID Discovery query. This link, defined in the [JRD section](#payid-easy-checkout-jrds) 
+of this specification, includes the PayID Easy Checkout URL representing a resource on the wallet which can
+be used to complete a payment.
+
+Payment receivers who wish to initiate an Easy Checkout flow MUST query the sender's PayID Discovery server to 
+obtain a PayID Easy Checkout URL. PayID servers that wish to enable PayID Easy
+Checkout MUST include a JRD Link conforming to the definition in the [JRD section](#payid-easy-checkout-jrds) of this paper 
+in all PayID Easy Checkout Discovery responses.
+
+Payment receivers SHOULD implement fallback measures to complete a checkout flow if a user's wallet does not support PayID Easy Checkout.
+
+The following steps describe how a PayID client can query a PayID server to obtain a PayID Easy Checkout URL. 
+
+### Step 1: Assemble PayID Easy Checkout Discovery URL
+The process of assembling a PayID Discovery URL is defined in section 4.1.1 of [PAYID-DISCOVERY][], and is
+the same as for this protocol.
+
+### Step 2: Query PayID Easy Checkout Discovery URL
+Querying the PayID Discovery URL is defined in section 4.1.2 of [PAYID-DISCOVERY][], and is performed
+in the same way as this protocol.
+
+Clients SHOULD implement fallback measures to complete checkout if the PayID Easy Checkout Discovery query fails.
+
+### Step 3: Parse PayID Easy Checkout Metadata
+If PayID Easy Checkout is supported, a PayID Discovery server MUST respond with an HTTP status code `200` and a JSON payload
```

Comment:
Couldn't you issue a redirect as well to another PayID Discovery server?

Reply:
Currently, the spec doesn't allow it, although it could.  I'm not really sure what the use case would be (I think I've advocated for removing it from the PayID Discovery spec) for delegating to another PayID Discovery Server.  I think we should restrict easy checkout discovery responses to Easy Checkout URL links until there is a need for more advanced delegation.  If/when that time comes, we can always add it to this spec, or define a new JRD link type in a new spec

Reply:
Currently, Interactive PayID Discovery only supports delegation on a whole-domain basis via [this link-relation](https://github.com/payid-org/rfcs/blob/master/dist/spec/payid-discovery.txt#L565).


Interactive-mode PayID DIscovery also [unintentionally makes a redirect a failure mode](https://github.com/payid-org/rfcs/blob/master/dist/spec/payid-discovery.txt#L406). Def something to revisit if Discovery becomes widely used.